### DISCUSSION TITLE
FlareSolverr LXC script error

### DISCUSSION THREAD

**Discussion by kokobadu** [asylumexp/Proxmox#72] [Q&A] [answered]
Many thanks for all the great work.

With FlareSolverr LXC script I found one error.

Script reports error on:
$STD apt-get install -y chromium=126.0.6478.126-1~deb12u1

While executing with default settings:

```
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 105
Using Hostname: flaresolverr
Using Disk Size: 4GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a FlareSolverr LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local for Container Storage.
 ✓ LXC Container 105 was successfully created.
 ✓ Started LXC Container
bash: warning: setlocale: LC_ALL: cannot change locale (nl_NL@euro)
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.1.180 2001:1c00:1f0b:f600:be24:11ff:fe25:34ff 
 ✓ IPv4 Internet Connected
 ✓ IPv6 Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 - Installing Dependencies   
[ERROR] in line 23: exit code 0: while executing command "$@" > /dev/null 2>&1
The silent function has suppressed the error, run the script with verbose mode enabled, which will provide more detailed output.
```

Executing with custom parameters leads to the same error.

Executing its origanal from @tteck on x64 Proxmox all works fine.

Perhaps someone can help.

**Comment by asylumexp** [comment]
Ill have a look now

**Comment by kokobadu** [comment]
Hi and thanks for look at it. I assume you changed something in the script, since now another error pops up:

`[ERROR] in line 45: exit code 0: while executing command wget https://github.com/electron/electron/releases/download/v31.1.0/chromedriver-v32.0.1-linux-arm64.zip -O /opt/flaresolverr/webdriver.zip`

Just fyi.

**Comment by kokobadu** [comment]
I belive it should be:

`wget https://github.com/electron/electron/releases/download/v32.0.1/chromedriver-v32.0.1-linux-arm64.zip -O /opt/flaresolverr/webdriver.zip`

**Comment by asylumexp** [comment]
Oops, saw the error when I tested it but had to do something meant to come back but forgot to, just fixed the link

**Reply by asylumexp** [reply, marked as answer]
fixed in https://github.com/asylumexp/Proxmox/commit/4b824b4d5f094c622fe63449966d9606e893e5a9
